### PR TITLE
Add hover effect to the AUTO refresh UI in the preview

### PR DESCRIPTION
**Pull request by hritwik567**
Color changes to dark grey when we hover over AUTO refresh button.

### CODE REVIEW

**Comment by gideonthomas** [comment]
Fixes #1723.

Similar fix available at https://github.com/mozilla/thimble.mozilla.org/pull/1733

File: public/editor/stylesheets/editor.less [hunk 1]
```diff
@@ -651,6 +651,10 @@ body {
         background: #EEE;
       }
     }
+    
+    &:hover {
+      color: rgba(0,0,0,.5);
```
Comment (flukeout) [inline public/editor/stylesheets/editor.less]: Please change to ``color: rgba(0,0,0,.6);`` to match the 'enabled' state and we'll be good to go.

**Comment by hritwik567** [comment]
@flukeout changed the rgba value as requested

**Comment by flukeout** [comment]
Great! Thanks a lot @hritwik567 - merging!

**Pull request merged**